src/main/java/cn/nukkit/plugin/JavaPluginLoader.java

```diff
@@ -64,13 +70,148 @@ public Plugin loadPlugin(File file) throws Exception {
                 }
 
             } catch (ClassNotFoundException e) {
-                throw new PluginException("Couldn't load plugin " + description.getName() + ": main class not found");
+                PluginAssert.findMainClass(description.getName());
             }
         }
 
         return null;
     }
 
+    //simple load Plugin
+    @Override
+    public Plugin simpleLoadPlugin(File file) {
```

Comment:
This is the core part of loading the simple plugin, initializing the plugin information.